### DISCUSSION TITLE
2023-07-24

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#129] [daily_meeting]
### 개인 일정
2시 ~ 2시 30분 : 노아 근로
3시 ~ 4시 : 노아, 룩소, 마코 근로 회의

### 오늘의 할 일
- UX 저자 워크숍 사전 미션(~7/25 화) -> 준이 제공한 내용 참고해 데모 끝나고 월요일에 하기로 결정
- 백엔드
  - 패키지 구조 수정으로 인한 리팩토링
    - 터지는 테스트는 내일 더 리팩토링해야 할 듯함.

### 화요일까지 할 일
- UX 저자 워크숍 사전 미션(~7/25 화) -> 준이 제공한 내용 참고해 데모 끝나고 월요일 진행
- 서비스 론칭 완료 및 피드백 반영사항 기능 구현 (~ 1주차 화요일)
    - 프론트, 백 미구현 기능 구현 및 리팩토링 진행
    - 프론트, 백 서비스 연동 완료
    - cloudFlare HTTPS 적용해서 서비스 배포
    - 배포 DB 드랍방지 설정
    - API 문서화 진행하기
    - 질문 한 두개만 빼고 선택으로 바꾸기
    - 정식 론칭 서비스 배포 완료 (HTTPS 적용)
    - 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
    - 시간 선택 범위 확대하기

### 언젠가 해야할 일
- 기획 변경된 이유 정리
- DNS 어떻게 할지

**Comment by aak2075** [comment]
### 체크인

10점 - 잠실캠 와서 너무 좋아요

### 체크아웃

10점 - 설계 이야기 하니까 재밌어요 다만 시간이 너무 빨리가네요ㅠ

**Comment by jaehee329** [comment]
### 체크인
8점 - 내 주말 어디갔지? 월요일이 오후부터 시작이라 정말 다행입니다

### 체크아웃
8점 - 무난무난한 월요일이네요. 괜찮았어요

**Comment by nlom0218** [comment]
### 체크인

10점 - 평일에 잠실이라니... 재미집니다. 자리 선정도 좋아서 만족합니다 👍

### 체크아웃

9점 - 월요일여서 그런가요. 한게 없는데 벌써 퇴근시간이네요 😭

**Comment by woosung1223** [comment]
### 체크인
9점 - 월요일이지만 오랜만에 크루들 얼굴 봐서 재밌습니당

### 체크아웃
9점 - 패키지 구조를 생각보다 빨리 바꿔서 좋네요 코드가 좋은 방향으로 가는게 느껴집니두

**Comment by woo-jk** [comment]
### 체크인
7점 - 시간 계산 잘못해서 택시탐 ㅠㅠ

### 체크아웃
7점 - 팀 피드백에 시간이 너무 오래걸려서 원래 하려던 개발을 못했다.

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 주말에 알차게 시간을 보낸 것 같아서 기분이 좋은데 잠실까지 와서 더 좋네요!

### 체크아웃
8점 - 살짝 피곤해졌지만 그래도 가장 큰 문제였던 패키지 구조를 거의 완료해가는 것 같아서 기분이 좀 커버가 된 것 같습니다!
